docs/user-guide/sdh/release-hub.md

```diff
@@ -0,0 +1,265 @@
+
+    ![Figure 9: Choosing Applications](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/search-apps.gif)
+
+4. Click **Add Release Stage**.
+
+    ![Figure 10: Dividing Release in Stages](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/add-release-stage.jpg)
+
+{% hint style="info" %}
+### Importance of Release Stages
+By default, your selected applications will be set to release in one go. However, you can also release them in stages. In other words, you can decide which set of applications should be released first, subsequently which ones to release next, and the ones to release last. <br /><br /> For example, if you have a new login feature (backend) and a dashboard update (frontend), you might want to release the login feature first to ensure users can log in before seeing the updated dashboard. 
+{% endhint %}
+
+5. Use the drag-and-drop feature to move applications from one stage to another.
+
+    ![Figure 11: Rearranging the Sequence](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/release-stage-v2.gif)
+
+{% hint style="warning" %}
+The drag-and-drop feature is designed specifically for moving applications between different release stages. It is not meant to alter the sequence of applications within the same stage.
+{% endhint %}
+
+6. Once you have finalized the sequence and stages, click **Save Changes**.
+
+---
+
+## Selecting Images
+
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to select images for selected applications.
+{% endhint %}
+
+1. Select a workflow available for your application. All the [images](../../reference/glossary.md#image) available in the selected workflow will appear.
+
+    ![Figure 12: Selecting Image from Specific Workflow](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/workflow-selection.gif)
+
+2. Click **SELECT** next to the image you wish to deploy from the list.
+
+3. Repeat the above steps for other applications you added in the release.
+
+    ![Figure 13: Repeating Steps for Other Applications](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/image-selections.gif)
+
+4. Click **Save**.
+
+{% hint style="info" %}
+### Tip
+In case you don’t have the correct images ready for any of your applications, you can partially save your changes at this point, and return once they are ready.
+{% endhint %}
+
+5. You may add release instructions for each application using the in-built Markdown editor. This can be detailed deployment notes and configuration guidelines for the team.
+
+    ![Figure 14: Adding Release Instructions](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/sdh/release-instructions-v2.gif)
+
+6. Before freezing the requirements, make sure the release order is correct, add applications if needed, and include environments in tenants (if not done already). Once you have finalized them, click **Lock Requirements**. 
```

Comment:
```suggestion
6. Before locking the requirements, make sure the release order is correct, add applications if needed, and include environments in tenants (if not done already). Once you have finalized them, click **Lock Requirements**. 
```

Reply:
Will pick this up in the next iteration of this doc

Reply:
Fixed...committed in this iteration itself 👍